storage/innobase/log/log0meb.cc

```diff
@@ -2323,12 +2324,18 @@ long long innodb_redo_log_consumer_register(
     [[maybe_unused]] UDF_INIT *initid, [[maybe_unused]] UDF_ARGS *args,
     [[maybe_unused]] unsigned char *null_value,
     [[maybe_unused]] unsigned char *error) {
+  std::string name = "MEB";
   if (current_thd == nullptr ||
       verify_privilege(current_thd, backup_admin_privilege)) {
     return 1;
   }
-  return static_cast<long long>(
-      meb::redo_log_consumer_register(thd_to_innodb_session(current_thd)));
+
+  if (args->arg_count == 1) {
```

Comment:
I would use `>=` here. Right now, we error out if the argument count is greater than 1, but later, if for some reason we do add more arguments, `>= 1` will still work.